src/test/compile-fail/const-eval-span.rs

```diff
@@ -14,7 +14,7 @@
 struct S(i32);
 
 const CONSTANT: S = S(0);
-//~^ ERROR: constant evaluation error: non-constant path in constant expression [E0080]
+//~^ ERROR: constant evaluation error: call on struct [E0080]
```

Comment:
just an artifact. it works in `trans/consts`, but not in `const_eval`. It's in the queue after fixing tuples and structs.